ui/pages/confirmation/templates/switch-ethereum-chain.js

```diff
@@ -46,9 +47,10 @@ function getValues(pendingApproval, t, actions) {
         children: t('switchEthereumChainConfirmationDescription'),
         props: {
           variant: TYPOGRAPHY.H7,
+          color: COLORS.UI4,
```

Comment:
I believe this should be updated to use our new colors from the design system so that dark mode and other themes in the future will work appropriately. @georgewrmarshall could you advise on the best way to future proof this new work?